adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -2204,6 +2206,34 @@ trait DatasetBoundGenomicDataset[T, U <: Product, V <: GenomicDataset[T, U, V]]
     transformDataset(_.unpersist())
   }
 
+  val isPartitioned: Boolean
+  val optPartitionBinSize: Option[Int]
+  val optLookbackPartitions: Option[Int]
+
+  private def referenceRegionsToDatasetQueryString(regions: Iterable[ReferenceRegion], partitionSize: Int = 1000000, partitionedLookBackNum: Int = 1): String = {
```

Comment:
shouldn't this method use the vals `optPartitionBinSize` and `optLookbackPartitions` above?

Reply:
eeek, yes you are correct.

Reply:
Well actually, it was working correctly before because the optPartitionedBinSize was what was passed as a parameter when called fro `filterByoverlappingRegions` but given that both functions are in the same trait, no no need for the parameter.

Reply:
fixed